learning_journal/security.py

```diff
@@ -0,0 +1,46 @@
+import os
+from passlib.apps import custom_app_context as pwd_context
+from pyramid.authentication import AuthTktAuthenticationPolicy
+from pyramid.authorization import ACLAuthorizationPolicy
+from pyramid.security import Allow, Authenticated, Everyone
+
+
+def includeme(config):
+    """Security configuration"""
+    auth_secret = os.environ.get('AUTH_SECRET', 'randomstring')
```

Comment:
It might be worth your while to ensure that the default behavior is to fail configuration if this SECRET is not present.